packages/css/src/components/progress-list/progress-list.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@use "../../common/hyphenation" as *;
+@use "../../common/text-rendering" as *;
+
+@mixin reset-ol {
+  list-style-type: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+}
+
+.ams-progress-list__connector {
+  background-color: var(--ams-progress-list-connector-background-color);
+  block-size: 100%;
+  grid-column: 1; // This is used to position the connector in the middle of the marker.
+  grid-row: 1; // This is used to position the connector in the middle of the marker.
+  inline-size: var(--ams-progress-list-connector-inline-size);
+
+  &-heading-2 {
+    margin-block-start: calc(
+      (var(--ams-progress-list-marker-heading-2-line-height) * var(--ams-progress-list-marker-heading-2-font-size)) / 2
+    );
+  }
+
+  &-heading-3 {
+    margin-block-start: calc(
+      (var(--ams-progress-list-marker-heading-3-line-height) * var(--ams-progress-list-marker-heading-3-font-size)) / 2
+    );
+  }
+
+  &-heading-4 {
+    margin-block-start: calc(
+      (var(--ams-progress-list-marker-heading-4-line-height) * var(--ams-progress-list-marker-heading-4-font-size)) / 2
+    );
+  }
+}
+
+.ams-progress-list {
+  @include hyphenation;
+  @include text-rendering;
+  @include reset-ol;
+
+  list-style: none;
+}
+
+.ams-progress-list__step {
+  display: flex;
+
+  &:last-child .ams-progress-list__connector {
+    display: none;
```

Comment:
We do need the part of the connector from the top edge of the step to the middle of the marker here.